server/database/config/db_build.js

```diff
@@ -0,0 +1,17 @@
+const fs = require('fs');
+const path = require('path');
+
+const dbConnection = require('./db_Connection');
```

Comment:
Please check the db connection name of the file it supposes to be like this "db_connection"

Reply:
done !